agent/proxy/manager_test.go

```diff
@@ -261,6 +262,50 @@ func TestManagerRun_daemonPid(t *testing.T) {
 	require.NotEmpty(pidRaw)
 }
 
+// Test to check if the parent and the child processes
+// have the same environmental variables
+
+func TestEnvironproxy(t *testing.T) {
```

Comment:
nit pick: I'd rename this to be a little more descriptive since this is the name that shows up in test output. We also tend to prefix with the system being tested - see other tests in file all start `TestManager*`

I suggest something like: `TestManagerPassesEnvironment`